pkg/cli/cvd.go

```diff
@@ -101,6 +106,7 @@ type CreateCVDOpts struct {
 	// If true, perform the ADB connection automatically.
 	AutoConnect               bool
 	BuildAPICredentialsSource string
+	CredentialFilePath        string
```

Comment:
Would it be better to combine these two strings in a struct? Because I think they would be used at the same time. For example:
```
type BuildAPICredentialsOpts struct {
    Source string
    FilePath string
}
```